features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/h2.sql

```diff
@@ -1804,3 +1804,33 @@ CREATE TABLE IF NOT EXISTS AM_USER (
     PRIMARY KEY(USER_ID)
 );
 -- End of API-MGT Tables --
+
+-- Create Endpoint Registry Tables --
+
+CREATE TABLE IF NOT EXISTS ENDPOINT_REG (
+  ID INTEGER AUTO_INCREMENT NOT NULL,
+  UUID VARCHAR(255) NOT NULL,
+  REG_NAME VARCHAR(255),
+  REG_TYPE VARCHAR(20),
+  MODE VARCHAR(20),
+  TENANT_ID INTEGER DEFAULT -1,
+  REG_OWNER VARCHAR(255),
+  READ_ROLE VARCHAR(20),
+  WRITE_ROLE VARCHAR(20),
+  UNIQUE (UUID),
+  PRIMARY KEY(ID)
+);
+
+CREATE TABLE IF NOT EXISTS REG_ENTRY (
```

Comment:
shall we rename to ENDPOINT_REG_ENTRY. to avoid confusion with our registry  tables

Reply:
done